components/atom/upload/src/index.js

```diff
@@ -30,19 +37,34 @@ const AtomUpload = ({
     setReady(true)
   }, [])
 
+  const handleFileSelectionChange = files => {
+    onFilesSelection(files)
+  }
+
   const renderStatusBlock = status => {
     const classNameIcon = `${BASE_CLASS}-icon${capitalize(status)}`
     const IconStatus = props[`icon${capitalize(status)}`]
     const textStatus = props[`text${capitalize(status)}`]
     return (
       <div className={cx(BASE_CLASS, `${BASE_CLASS}--${status}`)}>
-        <span className={classNameIcon}>{IconStatus}</span>
-        <div className={CLASS_BLOCK_TEXT}>
-          <h4 className={CLASS_BLOCK_TEXT_MAIN}>{textStatus}</h4>
-          {status === STATUSES.ACTIVE && textExplanation && (
-            <p className={CLASS_BLOCK_TEXT_SECONDARY}>{textExplanation}</p>
-          )}
-        </div>
+        <>
+          <AtomIcon
+            size={ATOM_ICON_SIZES.large}
+            color={ATOM_ICON_COLORS.currentColor}
+            className={classNameIcon}
+          >
+            {IconStatus}
+          </AtomIcon>
+          <div className={CLASS_BLOCK_TEXT}>
+            <h4 className={CLASS_BLOCK_TEXT_MAIN}>{textStatus}</h4>
+            {status === STATUSES.ACTIVE && (textExplanation || Button) && (
```

Comment:
Could we put in a more descriptive statement? It's a little confusing to me.

```js
const isActive = status === STATUSES.ACTIVE
const hasTextExplanation = Boolean(textExplanation)
const hasButton = Boolean(Button)

return (
    [...]
    {isActive && (hasTextExplanation || hasButton) && [...]}
)
```

Reply:
Done!